packages/client/src/components/TransactionList.js

```diff
@@ -190,17 +121,35 @@ function TransactionList({ safeData, transactions = [{ foo: "bar" }] }) {
     );
   };
 
-  return transactions.map((transaction, index) => {
-    return (
-      <TransactionListItem
-        key={transaction.flowTransactionId}
-        transaction={transaction}
-        displayIndex={transactions.length - index}
-        isLastInList={index === transactions.length - 1}
-        onView={onViewTransaction}
-      />
-    );
-  });
-}
+  // only show txs with intent
+  const transactionsToShow = transactions.filter(
+    tx => tx?.blockEventData?.actionView?.intent
+  );
+
+  return (
+    <div className="border-light rounded-sm">
+      <div className="is-flex has-text-grey border-light-bottom is-hidden-mobile">
```

Comment:
missing columns class?

Reply:
so `columns` actually has all this extra baggage that makes it easier to just not use it in most cases... i agree its weird but adding it here means I have to do a bunch of overrides as well lol

![image](https://user-images.githubusercontent.com/15314629/191599074-49df8842-0823-4850-9c63-1b83d8d6a88e.png)
![image](https://user-images.githubusercontent.com/15314629/191599120-c150406e-df61-4be4-84e6-370760b870bc.png)

Reply:
I see, would `columns is-gapless` help solving the problem?

Reply:
that adds this and a few other styles that are troublesome 

![image](https://user-images.githubusercontent.com/15314629/191608025-e222ba6f-b9ae-4be9-afdc-7e1ce7f97d80.png)

The reason for not using bulma `columns` is because it assumes things of equal width or in neat little fractions like 1/3 etc but really this is a table where most of the columns are always going to be same width except for the variable string "Info" column.